mojave/items/medical/stack_medical.dm

```diff
@@ -0,0 +1,21 @@
+//Children of base TG stack medical items should go here
+
+/obj/item/stack/medical/suture/ms13
+    name = "doctor's bag"
+    singular_name = "use" //This is so that examine text says "X uses left in the stack" instead of "X doctor's bags left in the stack"
+    desc = "A bag filled with medical tools and supplies for patching and stabilizing wounds from conventional sources."
```

Comment:
```suggestion
    desc = "A bag filled with medical tools and supplies for patching and stabilizing brute damage."
```

I think this may make things clearer.